templates/saved_songs.html

```diff
@@ -10,8 +10,8 @@ <h1>Saved Songs</h1>
                 {% for song in saved_songs %}
                     <li class="list-group-item d-flex justify-content-between align-items-center">
                         <div>
-                            <h5>{{ song['song_name'] }}</h5>
-                            <p>{{ song['artist_name'] }}{% if song['album_name'] %} - {{ song['album_name'] }}{% endif %}</p>
+                            <h5 style="color: black;">{{ song['song_name'] }}</h5>
```

Comment:
I'm sure this is something to come back to but add classes or ids to your html elements and make a css file for all your css styling. Inline styling should be avoided unless you're fighting a css framework which I dont believe you're using here.